### ISSUE TITLE
Recurring Queued Scans but No Uploads

### ISSUE THREAD

**Issue by who8mypnuts** [jotta/jotta-cli-issues#120]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):** 0.7.33634

**Description of problem:**
Jotta cli starts normally. It runs the scan, starts to upload. Then encounters an "upload error", usually on a different file every time. Then it stops uploading and just says: "Queueing scheduled scan of..." over and over every minute.

Alternative: I can stop the service `sudo systemctl stop jottad` and then start it up again, and it will pretend to upload and then presents with an error like the one below. So, at this point in time, files are being uploaded essentially wasting my bandwidth only to end with an error for every single file.

**Expected:**

**jotta-cli status (`jotta-cli status`):**
```
All three paths are "Waiting for scan to start"
```

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
```
pid:1088934 2020/10/29 15:36:05 Error uploading {"Name":"1. Section Introduction.mp4","Path":"/mnt/Business/Reference/Training/Web/Javascript/Algorithms in JavaScript   Design techniques/6. Backtracking/1. Section Introduction.mp4","Size":2429127,"Modified":"2020-10-25T21:24:49.4997444-04:00","Created":"2020-10-28T08:02:45.7457429-04:00","Checksum":"","State":"Added"} => 
XID: 
pid:1088934 2020/10/29 15:36:24 Queueing scheduled scan of /mnt/Ragets
pid:1088934 2020/10/29 15:36:44 Queueing scheduled scan of /mnt/Business
pid:1088934 2020/10/29 15:36:44 Queue already running on backup /mnt/Business
```

**Traceback**
```

```

**Additional info:**
This has been happening since around October 6th, prior to that time, everything worked smashingly.

**Comment by Kimbsen** [comment]
Thanks for the report. We are looking into this. It could help if you could send an email to support@jottacloud.com with some additional details, like your username and the jottabackup.log file.

**Comment by Kimbsen** [comment]
We've just released version 0.7.35160 which should fix this issue.
